keyboards/nightly_boards/octopadplus/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
Thanks @zvecr. I already made the changes